src/fairplay.js

```diff
@@ -0,0 +1,40 @@
+import { stringToArray } from './util';
+
+export default function concatInitDataIdAndCertificate(initData, id, certificate) {
+  if (typeof id === 'string') {
+    id = stringToArray(id);
```

Comment:
In general yes, though I wanted to follow existing format to be as close to original HTML5 implementation as possible.